client/src/components/squids/SquidForm.jsx

```diff
@@ -0,0 +1,136 @@
+/* eslint-disable jsx-a11y/no-noninteractive-element-to-interactive-role */
+import React, { useState } from "react"
+
+import { useForm } from "react-hook-form"
+import { useQueryClient, useMutation } from "react-query"
+
+import "../../style/form.pcss"
+import { ApiClient } from "../../backend/ApiClient"
+
+export const SquidForm = () => {
+  const {
+    register,
+    reset,
+    handleSubmit,
+    formState: { errors },
+  } = useForm()
+  const [submitMessage, setSubmitMessage] = useState("")
+
+  const resetForm = () => {
+    reset({
+      name: "",
+      species: "",
+      specialPower: "",
+      experiencePoints: "",
+    })
+    setSubmitMessage("")
+  }
+
+  const addSquidMutation = useMutation((formPayload) => {
```

Comment:
pattern wise, we create a `hooks` directory and extract these constants to separate files within that directory. I think you also want to explicitly `return` in the callback option